.travis_script.bash

```diff
@@ -9,9 +9,27 @@ then
   exit 0
 fi
 
+download_changelog_tool(){
+  echo "Downloading and building pony stable..."
+
+  git clone "https://github.com/jemc/pony-stable"
+  pushd pony-stable && make && sudo make install
+  popd
+
+  echo "Downloading and building the changelog tool..."
+
+  git clone "https://github.com/ponylang/changelog-tool"
+  pushd changelog-tool && make && sudo make install
```

Comment:
I'm worried about this. it means we need to make sure that we can always build with master from changelog-tool. i think this should check out a specific tagged version. then we can update the version as needed.